### DISCUSSION TITLE
How to block by ip address?

### DISCUSSION THREAD

**Discussion by roysG** [emqx/emqx#6623] [Q&A] [answered]
I understand how to block by clientID and username, but i did not see any option to block by ip address,
Does it possible?

**Comment by zmstone** [comment, marked as answer]
Hi @roysG you can find examples in the doc
e.g. acl-file: https://docs.emqx.io/en/broker/v4.3/advanced/acl-file.html#define-acl

**Reply by roysG** [reply]
Thanks
